docs/medical-api/api-reference/document/get-document.mdx

```diff
@@ -1,28 +1,20 @@
 ---
 title: "Get Document"
-description: "Downloads the Document contents of the specified URL."
-api: "GET /medical/v1/document/download"
+description: "Fetches the document and sends a presigned URL"
```

Comment:
Is this true? We don't fetch the doc here, just return a URL.

Would this be more accurate? `Gets a presigned URL for downloading the specified document.`